e2e/support/test_support.go

```diff
@@ -2245,7 +2253,7 @@ func OperatorPod(ns string) func() *corev1.Pod {
 		if err := TestClient().List(TestContext, &lst,
 			ctrl.InNamespace(ns),
 			ctrl.MatchingLabels{
-				"camel.apache.org/component": "operator",
+				"camel.apache.org/component": componentLabelValue,
```

Comment:
nitpick: as a further cleanup, we should probably deprecate some of the `camel.apache.org` annotation in favor of k8s's [recommended labels ](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/)